### DISCUSSION TITLE
Duda sobre proyecto

### DISCUSSION THREAD

**Discussion by felipesanchez3196** [rramosp/ai4eng.v1#153] [UDEA Students]
Buenos días profe, le quería preguntar que tal veía esta competencia donde hay que predecir las ventas diarias de varios productos.

Ventas en tiendas - Pronóstico de series temporales:

https://www.kaggle.com/competitions/store-sales-time-series-forecasting/overview

Y este otro, es directamente de machine learning:

https://www.kaggle.com/competitions/tabular-playground-series-may-2022/overview

Gracias!!

**Comment by rramosp** [comment]
ambos están ok ... pero tienes que fijarte si cumplen las condiciones de
número de datos etc, y si no, proponer alguna solución en las líneas que se
sugieren en la página del curso

On Thu, Jun 30, 2022 at 11:56 AM felipesanchez3196 ***@***.***>
wrote:

> Buenos días profe, le quería preguntar que tal veía esta competencia donde
> hay que predecir las ventas diarias de varios productos.
>
> Ventas en tiendas - Pronóstico de series temporales:
>
>
> https://www.kaggle.com/competitions/store-sales-time-series-forecasting/overview
>
> Y este otro, es directamente de machine learning:
>
>
> https://www.kaggle.com/competitions/tabular-playground-series-may-2022/overview
>
> Gracias!!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/153>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIJB5OLVJSWL5LGJFX3VRXGSJANCNFSM52KAVHVQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
